crates/ruff_cli/src/printer.rs

```diff
@@ -389,14 +389,57 @@ fn num_digits(n: usize) -> usize {
         .max(1)
 }
 
-/// Return `true` if the [`Printer`] should indicate that a rule is fixable.
-const fn show_fix_status(autofix_level: flags::FixMode) -> bool {
-    // If we're in application mode, avoid indicating that a rule is fixable.
-    // If the specific violation were truly fixable, it would've been fixed in
-    // this pass! (We're occasionally unable to determine whether a specific
-    // violation is fixable without trying to fix it, so if autofix is not
-    // enabled, we may inadvertently indicate that a rule is fixable.)
-    !autofix_level.is_apply()
+/// Return the number of safe ([`Applicability::Automatic`]) and unsafe ([`Applicability::Suggested`])
```

Comment:
Hmm... I'm not convinced by `unsafe` because it now results in mixed up terminology (should you use "unsafe" or suggested). I think we should either explicitly use suggested or unsafe (also externally to avoid confusion). @zanie what do you think?

Reply:
I'm leaning towards "suggested" for overall coherence. I'll try to take some time tomorrow to play with it and see how different options feel.

Reply:
Cool! Excited to hear your thoughts

Reply:
@MichaReiser did you guys decide on a way forward here?

Reply:
Resolving, waiting for @zanieb's clarification